e2e/README.md

```diff
@@ -0,0 +1,44 @@
+### Contents:
+1. [Prepare test environment](##-1.-Prepare-test-environment)
+2. [Prepare test data](##-2.-Prepare-test-data)
+3. [Running tests](##-3.-Running-tests)
+4. [FAQ](##-FAQ)
+
+### 1. Prepare test environment
+##### 1.1. Set up local Rocket Chat server
+* Install Rocket Chat meteor app by following this [guide](https://rocket.chat/docs/developer-guides/quick-start).
+
+##### 1.2. Set up detox
+* Install dependencies by following this [guide](https://github.com/wix/Detox/blob/master/docs/Introduction.GettingStarted.md#step-1-install-dependencies) (only Step 1).
+
+### 2. Prepare test data
+* Run Rocket Chat meteor app: `meteor npm start` (make sure you to run this command from project that you created on Step 1.1.).
+* Open `localhost:3000` in browser.
+* Sign up as admin.
+* Create public room `detox-public`.
+* Create user with role: `user`, username: `detoxrn`, email: `detoxrn@gmail.com`, password: `123`.
+* Create user with role: `user`, username: `diego.mello`, email: `diego.mello@rocket.chat`, password: `123`.
```

Comment:
Update all credentials to be generic.
I don't want my email on all tests 😛
It should be something like `YOUR_TEST_EMAIL@EMAIL.COM` and then you ask to update `./e2e.data` with same info.